rust/theoros/src/services/indexer/mod.rs

```diff
@@ -60,10 +60,10 @@ impl IndexerService {
         hyperlane_mailbox_address: Felt,
         hyperlane_validator_announce_address: Felt,
         pragma_feeds_registry_address: Felt,
-        starting_block: u64,
+        current_block: u64,
     ) -> Result<Self> {
         let stream_config = Configuration::<Filter>::default()
-            .with_starting_block(starting_block)
+            .with_starting_block(max(1, current_block.saturating_sub(START_INDEXER_DELTA)))
```

Comment:
We can start at zero no?
```suggestion
            .with_starting_block(max(0, current_block.saturating_sub(START_INDEXER_DELTA)))
```

Reply:
ah maybe, was not sure, edited